sources/packages/backend/libs/services/src/students-assessments/assessment-sequential-processing.service.ts

```diff
@@ -354,19 +354,24 @@ export class AssessmentSequentialProcessingService {
           completedStudentAssessmentStatus: StudentAssessmentStatus.Completed,
         },
       )
+      .groupBy("offering.offeringIntensity")
       .getRawOne<{
         [WorkflowOutputType.FederalFSC]: string;
         [WorkflowOutputType.ProvincialFSC]: string;
         [WorkflowOutputType.ScholarshipsBursaries]: string;
         [WorkflowOutputType.SpouseContributionWeeks]: string;
-        [WorkflowOutputType.BookCost]: string;
+        [WorkflowOutputType.BooksAndSuppliesCost]: string;
         [WorkflowOutputType.ReturnTransportationCost]: string;
+        offeringIntensity: OfferingIntensity;
       }>();
 
-    return Object.keys(WorkflowOutputType).map((key) => ({
-      workflowOutput: key as WorkflowOutputType,
-      total: +totals[key] || 0,
-    }));
+    return totals
+      ? Object.keys(WorkflowOutputType).map((key) => ({
+          workflowOutput: key as WorkflowOutputType,
+          total: totals ? +totals[key] : 0,
+          offeringIntensity: totals?.offeringIntensity,
+        }))
+      : null;
```

Comment:
Could you change this as below to make sure only values that are present will be potentially returned or is there a requirement to always return those?
```ts
const totalsResult: ProgramYearWorkflowOutputTotal[] = [];
Object.keys(WorkflowOutputType).forEach((key) => {
  const total = totals[key];
  if (total) {
    totalsResult.push({
      workflowOutput: key as WorkflowOutputType,
      total: +total,
      offeringIntensity: totals.offeringIntensity,
    });
  }
});
return totalsResult;
```